src/makeCalculator.js

```diff
@@ -4,7 +4,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
+    result: 0,
+
+    add(x) {
+      calc.result += x;
+    },
+    subtract(x) {
+      calc.result -= x;
+    },
+    multiply(x) {
+      calc.result *= x;
+    },
+    divide(x) {
+      calc.result = x !== 0 ? (calc.result /= x) : (calc.result = 0);
```

Comment:
The `divide` method currently sets the result to 0 when dividing by zero. This might not be the best way to handle division by zero. Consider throwing an error or returning a special value to indicate that division by zero is not allowed.